mantis-testcontainers/src/test/java/TestContainerHelloWorld.java

```diff
@@ -158,6 +158,7 @@ public static void prepareControlPlane() throws Exception {
                 .withNetwork(network)
                 .withNetworkAliases(CONTROL_PLANE_ALIAS)
                 .withExposedPorts(CONTROL_PLANE_API_PORT, DEBUGGER_PORT)
+                .withLogConsumer(out -> log.info("[Control Plane] {}", out.getUtf8String()))
```

Comment:
happy to revert this, but I did repeatedly need these logs for debugging the failing test.